snet_cli/config.py

```diff
@@ -19,11 +19,13 @@ def persist():
     conf["network.mainnet"] = {"default_eth_rpc_endpoint": "https://mainnet.infura.io"}
     conf["network.ropsten"] = {"default_eth_rpc_endpoint": "https://ropsten.infura.io"}
     conf["network.rinkeby"] = {"default_eth_rpc_endpoint": "https://rinkeby.infura.io"}
+    conf["ipfs"] = {"default_ipfs_endpoint": "http://ipfs.singularitynet.io:80"}
     conf["session"] = {
         "init": "1",
         "default_gas_price": "1000000000",
         "default_wallet_index": "0",
-        "default_eth_rpc_endpoint": "https://kovan.infura.io"
+        "default_eth_rpc_endpoint": "https://kovan.infura.io",
+        "default_ipfs_endpoint": "http://ipfs.singularitynet.io:80"
```

Comment:
It looks like everywhere this is used, it's pulled from just conf["ipfs"]. Is there a reason to add it as a session variable rather than just reading it in from the static config section?

Reply:
Removed from session